libs/libarchfpga/src/physical_types.h

```diff
@@ -662,8 +681,25 @@ struct t_capacity_range {
     }
 };
 
-/** Describes the possible placeable blocks within a physical tile type.
- *  A sub tile adds flexibility in the tile composition description.
+/**
+ * @brief Describes the possible placeable blocks within a physical tile type.
+ *
+ * Heterogeneous blocks:
+ *
+ * The sub tile allows to have heterogeneous blocks placed at the same grid location.
+ * Heterogeneous blocks are blocks which do not share neither the same functionality nor the
+ * IO interface, but do share the same (x, y) grid location.
+ * For each heterogeneous block than, there should be a corresponding sub tile to enable
+ * its placement within the physical tile.
+ *
+ *
+ * Equivalent sites:
+ *
+ * Moreover, the same sub tile enables to allow the placement of different implementations
+ * of a logical block.
+ * This means that two blocks that have different internal functionalities, but the IO interface of one block
+ * is a subset of the other, they can be placed at the same sub tile location within the physical tile.
+ * These two blocks can be identified as equivalent, hence they can belong to the same sub tile.
  */
 struct t_sub_tile {
     char* name = nullptr;
```

Comment:
explain equivalent_sites (below).  These are really the types of logical blocks (i.e. blocks types in the netlist) which could be placed in this type of subtile (I think).

Reply:
Explain capacity below.  Give how many of these sub_tiles there are?  (i.e. capacity_range of 4 .. 7 means we could place compatible (logical) netlist blocks at sub_tile indices 4 to 7 in the grid?

Reply:
Done